### DISCUSSION TITLE
Should Firefox shim be deleted?

### DISCUSSION THREAD

**Discussion by letavocado** [katspaugh/wavesurfer.js#2809] [Q&A] [answered]
Mentioned in comment date is expired 

// Firefox shim until 2023.04.11

https://github.com/katspaugh/wavesurfer.js/blob/c8ff5a3dd4d7603dea86785b53cad3ad62707f41/src/renderer.ts#L278-L279

**Comment by katspaugh** [comment, marked as answer]
Yeah, I've already removed it on a branch I'm working on.
